features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/apis/v1/validations_api_router.jag

```diff
@@ -1,96 +1,143 @@
+            var mobileValidator = new mobileValidatorModule.Validator(null);
+            var result = mobileValidator.validateForm(formName, queryParams, partialValidation);
+
+            if (result.status != mobileValidatorModule.ValidationStatus.VALID) {
+                response.status = 422; // Unprocessable Entity (WebDAV; RFC 4918)
+                response.contentType = "application/json";
+                var content = {
+                    status: "Validation Error",
+                    validationResults: result.validationResults
+                };
+                response.content = stringify(content);
+                return;
+            }
+            response.status = 200;
+            response.content = stringify({status: "Success"});
+        }
+    });
 
-});
-    
     routeManager.register('POST', 'publisher', '/publisher/api/validations/assets/webapp/overview_context', function (context) {
         var contextVal = context.post.overview_context;
 
         try {
-
             var oldContextVal = '';
             var isContextExists = isContextExist(contextVal, oldContextVal);
             print(isContextExists);
-
-        }
-        catch (e) {
-            log.debug('An asset of type: ' + shortName + ' could not be created.The following exception was thrown: ' + e);
-            response.sendError(500, 'An asset of type: ' + shortName + ' could not be created.Please check the server logs.');
+        } catch (e) {
+            log.error('An asset of type: ' + shortName
+                      + ' could not be created.The following exception was thrown');
+            log.error(e);
+            response.sendError(500, 'An asset of type: ' + shortName
+                                    + ' could not be created.Please check the server logs.');
         }
 
     });
 
+    routeManager.register('GET', 'publisher', '/publisher/api/validations/assets/{type}/{fieldName}/{fieldValue}', function (context) {
 
-    /**
-* Validate whether the context already exists
-*/
-function isContextExist(fieldValue,oldContext){
-    return apiProvider.isContextExist(fieldValue,oldContext);
-}
+        if (context.params) {
+            var shortName = context.params.type;
+            var fieldName = context.params.fieldName;
+            var fieldValue = context.params.fieldValue;
+        }
 
+        try {
+            var contextVal = '/' + fieldValue;
+            var oldContextVal = '';
+            var isContextExists = isContextExist(contextVal, oldContextVal);
+            print(isContextExists);
 
+        } catch (e) {
+            log.error('An asset of type: ' + shortName
+                      + ' could not be created.The following exception was thrown');
+            log.error(e);
+            response.sendError(500, 'An asset of type: ' + shortName
+                                    + ' could not be created.Please check the server logs.');
```

Comment:
"Please check the server logs." The end user will not be able to view server logs.